scenario/features/blockevents.feature

```diff
@@ -77,3 +77,23 @@ Feature: Block event listening
             | block                  |
             | filtered block         |
             | block and private data |
+
+    Scenario Outline: Checkpoint of block events
+        Given I create a checkpointer
+        And I use my checkpointer to listen for <type> events
```

Comment:
All the other steps use "a"/"an" or "the", not "my". It's a small thing but I think it's worth being consistent.
```suggestion
        And I use the checkpointer to listen for <type> events
```
Remember the code step definitions will need a matching change in all three languages too.